### PR TITLE
tests: add longer jest timeout to migrate test, set other timeouts to 20000

### PR DESCRIPTION
**Pull request by Loonride**
tests

**Did you add tests for your changes?**

Yes

**If relevant, did you update the documentation?**

No

**Summary**

The latest run of the migrate tests seems to have timed out: https://github.com/webpack/webpack-cli/runs/622211039

We should have a longer timeout

**Does this PR introduce a breaking change?**

No

**Other information**

### CODE REVIEW

File: test/migrate/config/migrate-config.test.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ const outputPath = path.join(__dirname, outputDir);
 const outputFile = `${outputDir}/updated-webpack.config.js`;
 const outputFilePath = path.join(__dirname, outputFile);
 
+jest.setTimeout(200000);
```
Comment (evilebottnawi) [inline test/migrate/config/migrate-config.test.js:14]: Let's use `20000` otherwise CI really can freeze and we should wait a lot of time

Comment (Loonride) [reply test/migrate/config/migrate-config.test.js]: I copied and pasted from another test not realizing that they are all 200,000 by mistake.

**Comment by Loonride** [comment]
I think `init-inquirer.test.js` needs long timeout since it installs dependencies

**Comment by evilebottnawi** [comment]
hm, maybe you are right, try to setup it on `1` minute

**Comment by rishabh3112** [comment]
Feel free to open if you feel there is some usecase for it now :)

**Pull request closed without merging**